next-i18next.config.js

```diff
@@ -1,6 +1,13 @@
 module.exports = {
   i18n: {
-    defaultLocale: "en",
-    locales: ["en", "id"],
+    defaultLocale: 'en',
+    locales: ['en', 'id'],
   },
+  react: {
+    useSuspense: false,
+  },
+  localePath: typeof window === "undefined" ?
```

Comment:
@adrai Can I have your thoughts on this ?

Reply:
Why does this needs to be done?

Reply:
The localePath for monorepos.

The typeof window: 

A defensive way.  Now that next-i18next config is imported in a client side page (_app.tsx), I'm not sure I would rely on webpack/nextjs to ensure it's dropped. I've been hit few times on older nextjs versions (next-i18next supports next > 10)

But what I'm not sure

- Is localePath used/interpreted by i18next on the client side ? or only on server side ? (I mean by default)

Reply:
https://github.com/i18next/next-i18next/issues/1552#issuecomment-981156476

both:

here on server side: https://github.com/i18next/next-i18next/blob/master/src/config/createConfig.ts#L128 (for i18next-fs-backend)
here on client side: https://github.com/i18next/next-i18next/blob/master/src/config/createConfig.ts#L194 (for backends used via use option, for example i18next-http-backend etc... [here an example](https://github.com/i18next/i18next-http-backend/blob/master/example/next/next-i18next.config.js#L22))

Reply:
Great ! So actually it seems it makes sense to test typeof window...

Seems confirmed by https://github.com/i18next/next-i18next/issues/1552#issuecomment-991786093